test/e2e/pod_create_test.go

```diff
@@ -1134,4 +1134,18 @@ ENTRYPOINT ["sleep","99999"]
 		Expect(session.OutputToString()).Should(ContainSubstring("/vol2"))
 	})
 
+	It("podman pod create --shm-size", func() {
```

Comment:
just for `run --ipc=host` on a pod that has `--shm-size` set right? there is no equivalent on the pod level I do not think

Reply:
Hm. I think that would work, since it'd just ignore the pod SHM settings. If the pod can't set `--ipc=host` then verifying a container with `--ipc=host` can join the pod should be tested